src/core/modules/auth/core-auth.controller.ts

```diff
@@ -30,14 +38,35 @@ export class CoreAuthController {
    * Logout user (from specific device)
    */
   @ApiOperation({ description: 'Logs a user out from a specific device' })
+  @ApiQuery({ description: 'If all devices should be logged out,', name: 'allDevices', required: false, type: Boolean })
+  @ApiOkResponse({ type: Boolean })
+  @ApiUnauthorizedResponse({
+    schema: {
```

Comment:
@NicoKaempf The schema description seems to be the same for all `ApiUnaouthorizedResponse`. Please check whether a shorter, standardized declaration is possible, e.g. by defining the schema object in one place and then including it or providing a specific decorator.